main/inc/lib/usermanager.lib.php

```diff
@@ -5108,4 +5108,34 @@ public static function getExpendedTimeInCourses($userId, $courseCode, $sessionId
         return 0;
     }
 
+    /**
+     * Get the boss user ID from a followed user id
+     * @param $userId
+     * @return bool
+     */
+    public static function getStudentBoss($userId)
+    {
+        $userId = intval($userId);
+        if ($userId !== 0) {
```

Comment:
should be > 0 (given session ID is a int unsigned, it doesn't hold negative values)